docs-ref-conceptual/get-started-tutorial-1-prepare-environment.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Configure your Azure CLI environment | Microsoft Docs
+description: Learn Azure CLI installation, sign in, setting environment variables, and creating Azure resources containing a random ID.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, az account, az group, az config, az init, configuration
+---
+# Prepare your environment for the Azure CLI
+
+In this tutorial step, you execute your first Azure CLI commands! This includes finding and setting your active subscription and setting default values. You also learn how to create resource groups containing a random ID to always guarantee a unique resource name.
```

Comment:
Suggestion "In this tutorial step, you'll execute your first Azure CLI commands!"

Reply:
Thanks for the suggestion, but @ MSFT our docs are written in the present tense.  We have a tool, Acrolinx, that actually asks us to remove "will".